lib/pure/net.nim

```diff
@@ -118,7 +118,7 @@ proc toOSFlags*(socketFlags: set[SocketFlag]): cint =
       result = result or MSG_PEEK
     of SocketFlag.SafeDisconn: continue
 
-proc newSocket(fd: SocketHandle, isBuff: bool): Socket =
+proc newSocket*(fd: SocketHandle, isBuff: bool): Socket =
```

Comment:
Could you change ``isBuff`` to ``buffered`` while you're at it, please? Also make it ``true`` by default :)